src/pages/Landing/index.tsx

```diff
@@ -155,3 +175,9 @@ export default () => {
     </Grid>
   );
 };
+
+
+function deleteCustomer() {
+  throw new Error('Function not implemented.');
+}
```

Comment:
Is this needed?  I think it can be deleted

Reply:
Yeah its needed for the declarationi for the deletecustomer if you want i can move it up so it can be more organzied